_styles/main.css

```diff
@@ -288,16 +288,70 @@ nav a:visited {
 }
 
 nav a:focus {
-    color: #08c;
-    fill: #08c;
```

Comment:
Please put these back in. We still want the left side links to change when in focus.

Reply:
Can't we have original logo color for icon when focused. If I do the same as mentioned all icon will have same color `#08c` and this can seem monotonous to users

Reply:
These won't effect the social logos on the right side because they get overwritten below, but this does effect the left side of the navbar. Without it, there is no feedback when a user hovers over "Store" or "Support."